install/upgrade.php

```diff
@@ -232,6 +231,22 @@ protected function upgrade_3_7() {
 		$this->empty_duplicated_strings_translations();
 	}
 
+	/**
+	 * Upgrades if the previous version is < 3.8.
+	 * Migrates language taxonomies from `polylang` option to `pll_language_taxonomies` option.
+	 *
+	 * @since 3.8
+	 *
+	 * @return void
+	 */
+	protected function upgrade_3_8() {
+		$options = get_option( 'polylang' );
+
+		if ( is_array( $options ) && isset( $options['language_taxonomies'] ) && is_array( $options['language_taxonomies'] ) ) {
+			update_option( 'pll_language_taxonomies', $options['language_taxonomies'] );
+		}
```

Comment:
You must ensure that this option is always created, with an empty array if necessary.
You also must endure that it's the case for new installs as we always do. The goal is to avoid a useless extra query to the options table.

Reply:
Done in 9d4ba73eeb7f207561be089b0c51f4ae860e8c02